### DISCUSSION TITLE
More teleport cooldown bypass permissions

### DISCUSSION THREAD

**Discussion by timr2000** [EssentialsX/Essentials#3981] [Ideas and Feature Suggestions] [answered]
Hello,
I've been struggling on my server because of the teleport cooldown. I don't want a cooldown on /spawn.
Can you add more bypass permissions? like:

essentials.teleport.cooldown.bypass.spawn
essentials.teleport.cooldown.bypass.tpa
essentials.teleport.cooldown.bypass.warp

**Comment by JRoy** [comment, marked as answer]
See #4010
